fab/inventory/staging-va

```diff
@@ -43,5 +43,7 @@ hqdb0-staging.internal-va.commcarehq.org
 hqdb0-staging.internal-va.commcarehq.org
 
 [riakcs]
+hqdjango4-staging.internal.commcarehq.org
```

Comment:
I think I should have put that up there before deploying riak to these machines the first time.  I had to manually bring down the riak cluster on the new machines and redeploy with these settings, so they would use `hqdjango4` as the control node and join the old cluster.